Zilliqa/README.md

```diff
@@ -96,17 +96,17 @@ Each of these category of transitions are presented in further details below:
 | Name | Params | Description | Callable when paused? |
 |--|--|--|--|
 |`Mint`| `to: ByStr20, amount : Uint128, initiator : ByStr20, to_bal : Uint128, current_supply : Uint128` | Mint `amount` number of new tokens and allocate them to the `to` address.  <br>  :warning: **Note:** 1) Minting is a privileged transition that can be invoked only by non-blacklisted minters, i.e., `initiator` must be a non-blacklisted `minter`, 2) Minting can only be done when the contract is not paused, 3) The `to` address cannot be blacklisted nor null. | <center>:x:</center> |
-|`Burn`| `amount : Uint128, initiator : ByStr20, initiator_balance : Uint128, current_supply : Uint128` | Burn `value` number of tokens.  <br>  :warning: **Note:**   1) Burning is a privileged transition that can be invoked only by non-blacklisted minters, i.e., `initiator` must be a non-blacklisted `minter`, 2) Burning can only be done when the contract is not paused.| <center>:x:</center>  |
+|`Burn`| `amount : Uint128, initiator : ByStr20, initiator_balance : Uint128, current_supply : Uint128` | Burn `amount` number of tokens.  <br>  :warning: **Note:**   1) Burning is a privileged transition that can be invoked only by non-blacklisted minters, i.e., `initiator` must be a non-blacklisted `minter`, 2) Burning can only be done when the contract is not paused.| <center>:x:</center>  |
```

Comment:
Ah all the `amount` change is due to our standardisation of using amount, good catch!